requirements.txt

```diff
@@ -8,4 +8,3 @@ open_clip_torch
 einops
 gradio<4.0.0
```

Comment:
Also note, some files like `net.py` import `torchvision` which is not listed in `requirements.txt`. Is it a bug?